src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Track is running");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck is stopped");
```

Comment:
The message in the `stopWork` method should also be more informative. Consider changing it to 'Truck has stopped working.' to clearly indicate the action.